packages/react-i18n/src/i18n.ts

```diff
@@ -102,6 +103,7 @@ export class I18n {
       country,
       pseudolocalize = false,
       onError,
+      loading,
     }: I18nDetails,
```

Comment:
Does this type have a `loading` property on it?

Reply:
Yeah, I added it here: https://github.com/Shopify/quilt/pull/771/files#diff-a87e8a7d929dffc3c1a320c5490eeee5R10

Reply:
Ah, I didn't see that. I'd rather it not be added there, because it doesn't actually exist there for the manager, and it's meant to represent the overall i18n "context" in which the app currently exists (whereas loading is a temporary state that applies to individual components). Instead, I'd rather augment this argument type (so, something like `I18nDetails & {loading?: boolean}`)

Reply:
Ah I see, will do 👍